global_helpers/panther_oss_helpers.py

```diff
@@ -163,121 +161,6 @@ def resource_lookup(resource_id: str) -> Dict[str, Any]:
-    Instead, use `from panther_detection_helpers.caching import put_string_set`."""
-    return caching.put_string_set(key=key, val=val, epoch_seconds=epoch_seconds)
-
-
-def add_to_string_set(key: str, val: Union[str, Sequence[str]]) -> Set[str]:
-    """Global `add_to_string_set` is DEPRECATED.
-    Instead, use `from panther_detection_helpers.caching import add_to_string_set`."""
-    return caching.add_to_string_set(key=key, val=val)
-
-
-def remove_from_string_set(key: str, val: Union[str, Sequence[str]]) -> Set[str]:
-    """Global `remove_from_string_set` is DEPRECATED.
-    Instead, use `from panther_detection_helpers.caching import remove_from_string_set`."""
-    return caching.remove_from_string_set(key=key, val=val)
-
-
-def reset_string_set(key: str) -> None:
-    """Global `reset_string_set` is DEPRECATED.
-    Instead, use `from panther_detection_helpers.caching import reset_string_set`."""
-    return caching.reset_string_set(key=key)
-
-
-def evaluate_threshold(key: str, threshold: int = 10, expiry_seconds: int = 3600) -> bool:
-    """Global `evaluate_threshold` is DEPRECATED.
-    Instead, use `from panther_detection_helpers.caching import evaluate_threshold`."""
-    return caching.evaluate_threshold(key=key, threshold=threshold, expiry_seconds=expiry_seconds)
-
-
-def check_account_age(key):
-    """Global `check_account_age` is DEPRECATED.
-    Instead, use `from panther_detection_helpers.caching import check_account_age`."""
-    return caching.check_account_age(key=key)
-
-
-def km_between_ipinfo_loc(ipinfo_loc_one: dict, ipinfo_loc_two: dict):
-    """
-    compute the number of kilometers between two ipinfo_location enrichments
-    This uses a haversine computation which is imperfect and holds the benefit
-    of being supportable via stdlib. At polar opposites, haversine might be
-    0.3-0.5% off
-    See also https://en.wikipedia.org/wiki/Haversine_formula
-    See also https://stackoverflow.com/a/19412565
-    See also https://www.sunearthtools.com/tools/distance.php
-    """
-    if not set({"lat", "lng"}).issubset(set(ipinfo_loc_one.keys())):
-        # input ipinfo_loc_one doesn't have lat and lng keys
-        return None
-    if not set({"lat", "lng"}).issubset(set(ipinfo_loc_two.keys())):
-        # input ipinfo_loc_two doesn't have lat and lng keys
-        return None
-    lat_1 = radians(float(ipinfo_loc_one.get("lat")))
-    lng_1 = radians(float(ipinfo_loc_one.get("lng")))
-    lat_2 = radians(float(ipinfo_loc_two.get("lat")))
-    lng_2 = radians(float(ipinfo_loc_two.get("lng")))
-    # radius of the earth in kms
-    radius = 6372.795477598
-    lng_diff = lng_2 - lng_1
-    lat_diff = lat_2 - lat_1
-
-    step_1 = sin(lat_diff / 2) ** 2 + cos(lat_1) * cos(lat_2) * sin(lng_diff / 2) ** 2
-    step_2 = 2 * atan2(sqrt(step_1), sqrt(1 - step_1))
-    distance = radius * step_2
-    return distance
-
-
 def geoinfo_from_ip(ip: str) -> dict:  # pylint: disable=invalid-name
```

Comment:
can we rename the `ip` argument to `ip_address_string` or `ip_str` in order to remove the pylint supress?